aodndata/moorings/products_handler.py

```diff
@@ -106,6 +108,31 @@ def _get_input_files(self):
         self.input_file_collection.download(self._upload_store_runner.broker, self.temp_dir)
         # TODO: Replace temp_dir above with cache_dir?
 
+    def _get_old_product_files(self):
+        """Get a list of the currently published aggregated_timeseries files for the site being processed."""
+
+        filter_list = [PropertyIsEqualTo(propertyname='site_code', literal=self.product_site_code),
+                       PropertyIsEqualTo(propertyname='data_category', literal='aggregated_timeseries')
+                       ]
+        wfs_features = self._get_wfs_features(filter_list, propertyname=['url'])
+
+        self.old_product_files = {}
+        var_pattern = re.compile(r'FV01_([A-Z0-9-]+)-aggregated')
+        for f in wfs_features:
```

Comment:
maybe better to report all files not just the first one? Same number of lines but doing more :)

```
urls = [x['properties']['url'] for x in wfs_features];
emsg =  "Could not determine variable of interest for "
invalids = [emsg+f"{url} for url in urls if var_pattern.search(url)]
if invalids:
    raise InvalidFileNameError('\n'.join(invalids))
else: 
    vmatches = [var_pattern.search(url) for url in urls]
    variables_of_interest = [var.group(1).replace('-','_') for var in vmatches]
```